src/ContinuityMassElemKernel.C

```diff
@@ -61,7 +61,7 @@ ContinuityMassElemKernel<AlgTraits>::ContinuityMassElemKernel(
   dataPreReqs.add_cvfem_volume_me(meSCV);
 
   // fields and data
-  dataPreReqs.add_gathered_nodal_field(*coordinates_, AlgTraits::nDim_);
+  dataPreReqs.add_coordinates_field(*coordinates_, AlgTraits::nDim_);
```

Comment:
Oh, I missed that in the add_coordinates_field() set of calls that I looked at in the Kernel.

I will look closer at fill_master_element(0 now to see how you can push in a coordinates view and still iterate over a map of ME calcs.